server/render/test/index.test.js

```diff
@@ -0,0 +1,249 @@
+/** @format */
```

Comment:
The filename should be `index.js` to be aligned with Calypso nomenclature (we rely on the `test/` subdir to indicate that this is a test).